### PR TITLE
Update & Improve Dutch translation

### PR DESCRIPTION
**Pull request by feikedonia**

### CODE REVIEW

File: i18n/nl/cosmic_edit.ftl [hunk 1]
```diff
@@ -1,27 +1,67 @@
+cosmic-text-editor = COSMIC-tekstbewerker
 new-document = Nieuw document
 open-project = Open project
+todo = Te doen
```
Comment (feikedonia) [inline i18n/nl/cosmic_edit.ftl:4]: Has to say: TODO

File: i18n/nl/cosmic_edit.ftl [hunk 2]
```diff
@@ -1,27 +1,67 @@
...
+syntax-light = Lichte syntaxmarkering
 default-font = Standaard lettertype
 default-font-size = Standaard lettertypegrootte
 
 ### Keyboard shortcuts
 keyboard-shortcuts = Toetsenbord snelkoppelingen
-enable-vim-bindings = Schakel Vim snelkoppelingen in
+enable-vim-bindings = Schakel Vim-snelkoppelingen in
+
+# Find
```
Comment (feikedonia) [inline i18n/nl/cosmic_edit.ftl:55]: Line 55 to 64 are based on the translation of the find-and-replace feature in Libreoffice

**Review by feikedonia — COMMENTED** [review]
Updated translation

**Review by feikedonia — COMMENTED** [review]
Changed: 
* Line 4: TODO
* Line 117: COSMIC-tekstbewerker

File: i18n/nl/cosmic_edit.ftl [hunk 3]
```diff
@@ -1,27 +1,67 @@
...
+## About
+git-description = Git commit {$hash} op {$date}
+
 ## Document statistics
 document-statistics = Document statistieken
 word-count = Aantal woorden
 character-count = Tekens
 character-count-no-spaces = Tekens (zonder spaties)
-line-count = Lijnen
+line-count = Regels
```
Comment (feikedonia) [inline i18n/nl/cosmic_edit.ftl:16]: The translation for 'lines of text' is 'tekstregels', so it should be 'regels'

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**